src/App.js

```diff
@@ -2,7 +2,44 @@ import React, { Component } from 'react';
 
 import './App.css';
 
+const BOARD_SIZE = 10;
+const PLAYER_QUANTITY = 2;
+const MAX_SHIP_PER_PLAYER = 10;
+const SHIP_TOUCHED = 1;
+const SHIP_MISS = 1;
+
 class App extends Component {
+
+    constructor() {
+        state = {
+            playerOneData = [{
+                isAllShipOnBoard: false,
+                shipPosition: [],
+                playerTwoReaminingShip : 0,
+                shipTouchedPosition: [],
+                shipMissedPosition: []
+            }],
+            playerTwoData = [{
+                isAllShipOnBoard: false,
+                shipPosition: [],
+                playerTwoReaminingShip : 0,
+                shipTouchedPosition: [],
+                shipMissedPosition: []
```

Comment:
Trouve un moyen de stocker dans le même tableau la data concernant this.shipPosition, this. shipTouchedPosition et this.shipMissedPosition.

Reply:
Okay je regarde et je pousse un truc :)